src/vs/workbench/contrib/comments/browser/commentsViewActions.ts

```diff
@@ -63,6 +70,16 @@ export class CommentsFilters extends Disposable {
 		}
 	}
 
+	private _sortBy = CONTEXT_KEY_SORT_BY.bindTo(this.contextKeyService);
+	get sortBy(): CommentsSortOrder {
+		return this._sortBy.get()!;
+	}
+	set sortBy(sortBy: CommentsSortOrder) {
+		if (this._sortBy.get() !== sortBy) {
+			this._sortBy.set(sortBy);
+			this._onDidChange.fire({ sortBy });
```

Comment:
The other options fire `true` no matter which value was actually set - should this do so too? I figured anything truthy would work, and it might be useful to be able to access the new value directly.